resources/definitions/vzbot_base.def.json

```diff
@@ -0,0 +1,165 @@
+{
+    "name": "VzBot Base",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": 
+    {
+        "visible": false,
+        "author": "Chris, ckvsoft.at",
+        "manufacturer": "VzBot",
+        "file_formats": "text/x-gcode",
+        "first_start_actions": ["MachineSettingsAction"],
+        "machine_extruder_trains": { "0": "vzbot_extruder_0" },
+
+        "has_machine_quality": true,
+        "has_materials": true,
+        "has_variants": true,
+        "variants_name": "Nozzle Size",
+        "preferred_quality_type": "standard",
+        "preferred_variant_name": "0.4mm Nozzle",
+        "preferred_material": "generic_abs",
+        "exclude_materials": [
+            "ultimaker_abs_black",
```

Comment:
You don't need to exclude these materials. They are 2.85 mm materials, so they are automatically filtered out.

Reply:
OK, I did not know that. Unfortunately I couldn't find any exact documentation. I copied that from the voron2. can i remove Thanks